tests/models/nn/test_sasrec.py

```diff
@@ -211,7 +215,7 @@ def test_u2i(
             batch_size=4,
             epochs=2,
             deterministic=True,
-            recommend_device=recommend_device,
+            recommend_accelerator=recommend_device,
```

Comment:
```suggestion
            recommend_accelerator=recommend_accelerator,
```
Also change the parameters of the test function

Reply:
fixed